src/styles/global-font-style.scss

```diff
@@ -59,4 +59,39 @@
 .font_navigator_logo {
   @include subtitle(600);
   line-height: 20px;
+}
+
+.font_hero_title {
+  @include h1-headline(700);
+  color: #fff;
+  text-shadow: 0px 4.1504716873168945px 21.897809982299805px rgba(0, 0, 0, 0.25);
```

Comment:
I am just curious why are there so many digits after the decimal point?

Reply:
Those numbers are copied from figma.

Reply:
> Those numbers are copied from figma.

So you did not attend and watch the QA session? There is no way to use a decimal number pixel in CSS.

Reply:
Round it to a closer even number.

Reply:
Solved

Reply:
I think the digits after the decimal point will affect very little to most of us. I searched Google that shows the majority of browsers now ignore the decimals.